.travis.yml

```diff
@@ -18,7 +18,7 @@ jobs:
       stage: Test (MacOS)
       os: 'osx'
       env: SCRIPT=test
-      node_js: 'stable'
+      node_js: 10
```

Comment:
Let's add comment here on issue :+1:

Reply:
Yeah, it was a good idea I saw it a bit late :) But I also don't have the URL before actually creating the PR, so :D Will update once the test-run is done.